CONTRIBUTING.md

```diff
@@ -0,0 +1,237 @@
+discussion and not be closed.
+
+### Adhere to Coding Style Guide
+
+We aim for a coherent and consistent code base, thus the coding style detailed in the 
+[styleguide](styleguide.md) should be followed.
+
+### Submit a Pull Request
+
+In addition to the contribution guideline made available in the 
+[Eclipse project handbook](https://www.eclipse.org/projects/handbook/#contributing),
+we would appreciate if your pull request applies to the following points:
+
+* Conform to [Pull-Request Etiquette](pr_etiquette.md)
+
+* Always apply the following copyright header to specific files in your work replacing the fields 
+  enclosed by curly brackets "{}" with your own identifying information. (Don't include the curly 
+  brackets!) Enclose the text in the appropriate comment syntax for the file format.
+
+    ```text
+    Copyright (c) {year} {owner}[ and others]
+
+    This program and the accompanying materials are made available under the
+    terms of the Apache License, Version 2.0 which is available at
+    https://www.apache.org/licenses/LICENSE-2.0
+
+    SPDX-License-Identifier: Apache-2.0
+
+    Contributors:
+      {name} - {description}
+    ```
+
+* The git commit messages should comply to the following format:
+    ```
+    <component>: <description>
+    ```
+
+  Use the [imperative mood](https://github.com/git/git/blob/master/Documentation/SubmittingPatches)
+  as in "Fix bug" or "Add feature" rather than "Fixed bug" or "Added feature" and
+  [mention the GitHub issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
+  e.g. `transfer process: improve logging, closes #3`.
+
+  All committers, and all commits, are bound to
+  the [Developer Certificate of Origin.](https://www.eclipse.org/legal/DCO.php)
+  As such, all parties involved in a contribution must have valid ECAs. Additionally, commits can 
+  include a ["Signed-off-by" entry](https://wiki.eclipse.org/Development_Resources/Contributing_via_Git).
+  
+* Add meaningful tests to verify your submission acts as expected.
+
+* Where code is not self-explanatory, add documentation providing extra clarification.
+
+* Add documentation files to new modules. See [here](#add-documentation) for more details.
+
+* Add relevant changes (e.g., no typo fixes, updated readme files, fixes of stuck test) to the 
+  [changelog](CHANGELOG.md). If these refer to a new feature, add this to the `Overview` section 
+  and add your changes to the `Detailed Changes` section according to the rules documented on 
+  <http://keepachangelog.com>. Include more information via linking to existing pull requests,
+  issues, or discussions.
+
+* If a new module has been added or a significant part of the code has been changed and you should 
+  or want to be seen as the contact person for any further changes, please add appropriate 
+  information to the [CODEOWNERS](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/main/CODEOWNERS) 
```

Comment:
link needs to be updated - currently it is pointing to EDC repo.

Reply:
updated link to point to current repo